src/executor/graph_executor.cc

```diff
@@ -947,8 +947,8 @@ void GraphExecutor::InitDataEntryMemory(std::vector<NDArray>* shared_pool) {
     for (uint32_t i = 0; i < idx[nid].source->num_outputs(); ++i) {
       auto eid = idx.entry_id(nid, i);
       data_context[eid] = vctx[nid];
-      CHECK_NE(vstorage_type[nid], kUndefinedStorage);
-      data_storage_type[eid] = (NDArrayStorageType) vstorage_type[nid];
```

Comment:
Thanks for the fix! Is there any failed unit test you have at hand?

Reply:
I'am very sorry for no response on this question. I found this problem just by reading code, and had no any test. I've no runtime enviroment at hand. And I have to admit your CI is too complicate for me. Can anyone help to tracking this problem?